api/envoy/config/core/v3/default_socket_interface.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
```

Comment:
I think this should live in some new socket interface extension tree in `api/envoy/extensions`.

Reply:
What do you think about `api/envoy/extensions/network/socket_interface/default_socket_interface.proto`? Or should we use socket instead of socket_interface?

Reply:
The versioned `api/envoy/extensions/network/socket_interface/v3/default_socket_interface.proto` is good (with respective package name). Thanks.

Reply:
Done!

Reply:
@htuch it seems the docs are complaining because we don't have an extension added to `source/extensions/extensions_build_config.bzl`. Is it okay to add one for network/socket_interface although we don't have any code beyond the proto file? Does this also mean it needs a maintainer in `CODEOWNERS`?

Reply:
nvm, removing the extension comment solved the issue

Reply:
@htuch, any idea if vhds test failure is a fluke? I only changed the location of `default_socket_interface.proto` and the PR verified several times until now.

Reply:
Yes, fluke, see https://github.com/envoyproxy/envoy/issues/9784.